impl/src/main/java/org/ehcache/impl/config/copy/DefaultCopierConfiguration.java

```diff
@@ -55,7 +55,7 @@ public DefaultCopierConfiguration(Copier<T> instance, Type type) {
     this.type = type;
   }
 
-  DefaultCopierConfiguration(Class<? extends Copier<T>> copierClass) {
+  public DefaultCopierConfiguration(Class<? extends Copier<T>> copierClass) {
```

Comment:
This constructor was not set as public for a good reason. Better not change that visibility and use a different constructor instead.